tests/system-test/1-insert/update_data.py

```diff
@@ -0,0 +1,223 @@
+###################################################################
```

Comment:
1.stb/ctb/nth
2.normal data->Null and Null -> normal data
3.binary/nchar increase length and update

Reply:
1-3 is OK，4-6 will  be fulfilled in another case